pkgs/dash_analytics/analysis_options.yaml

```diff
@@ -0,0 +1,34 @@
+# This file configures the static analysis results for your project (errors,
+# warnings, and lints).
+#
+# This enables the 'recommended' set of lints from `package:lints`.
+# This set helps identify many issues that may lead to problems when running
+# or consuming Dart code, and enforces writing Dart using a single, idiomatic
+# style and format.
+#
+# If you want a smaller set of lints you can change this to specify
+# 'package:lints/core.yaml'. These are just the most critical lints
+# (the recommended set includes the core lints).
+# The core lints are also what is used by pub.dev for scoring packages.
+
+include: package:lints/recommended.yaml
+
+# Uncomment the following section to specify additional rules.
+
+linter:
+  rules:
+    - always_declare_return_types
+    - always_specify_types
+    - camel_case_types
+    - prefer_single_quotes
+    - unawaited_futures
+
+# analyzer:
+#   exclude:
+#     - path/to/excluded/files/**
+
+# For more information about the core and recommended set of lints, see
+# https://dart.dev/go/core-lints
+
+# For additional information about configuring this file, see
+# https://dart.dev/guides/language/analysis-options
```

Comment:
@devoncarew do you see any lints that should be added or removed here?

Reply:
It's using package:lints/recommended.yaml, which is a great base. There are other things we could turn on - including experimenting with `package:dart_flutter_team_lints` - but I think we could experiment there after landing this PR.